pyspecProcScripts/QESR_calc.py

```diff
@@ -0,0 +1,104 @@
+"""
+Quantitative ESR
+================
+In this example, we baseline correct and
+integrate ESR data, and compare the result to
+the result of integration inside XEPR.
+
+This makes use of the package `pint`, which is a
+very nice package for handling units.
+"""
```

Comment:
To be completely honest, I don't know what this is - I think you and FS had done this together, but as of right now the only QESR scripts WK and I use are QESR_rescale.py and protein_QESR.py

Reply:
remove this file from the pull request